google-cloud-bigquery/lib/google/cloud/bigquery/table.rb

```diff
@@ -118,6 +118,16 @@ def table_ref
           table_ref
         end
 
+        ###
+        # The gapi fragment containing TimePartitioning ExpirationMS and Type
+        # as a camel-cased hash.
+        def time_partitioning
+          ensure_full_data!
+          time_part = @gapi.time_partitioning
+          time_part = time_part.to_hash if time_part.respond_to? :to_hash
+          time_part
```

Comment:
If this is public, should be documented following the style for public methods (e.g `#id` below)